Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -876,6 +877,11 @@ protected virtual void UpdateMenuItemIcon(Context context, IMenuItem menuItem, T
 				Drawable iconDrawable = context.GetFormsDrawable(icon);
 				if (iconDrawable != null)
 				{
+					if (!menuItem.IsEnabled)
+					{
+						iconDrawable.Mutate().SetAlpha(127);
```

Comment:
Why did we pick 127?

Also can you refactor this to a named const please?

Reply:
127 was what most of the internet agreed on, as well as looked pretty similar to the text ones when I tested. 

It can be any value we would like, or different for light luminosity vs dark or something. 

I just struggled to find a Google guideline for it, and tried different values and compared platforms till I found what looked similar. 

Also, you want that constant in this class, correct? Just making sure I put it where you want.